src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx

```diff
@@ -1,109 +1,70 @@
 import styled from "@emotion/styled";
-import { mixin, vars } from "../../../styles";
-import { lighten } from "polished";
+import tw from "twin.macro";
+import { mixin } from "../../../styles";
 
 export const ViewerContainer = styled.div`
-  text-align: center;
-  padding: 24px;
-  justify-content: center;
-  flex-direction: column;
-  display: flex;
-  border-radius: 10px;
-  min-height: 400px;
-
-  @media only screen and (min-width: ${vars.lg}) {
-    min-height: 600px;
-  }
+  ${tw`text-center p-6 flex flex-col justify-center rounded-xl min-h-400 lg:min-h-600`}
 
   &.default {
-    ${mixin.dropzoneStatus({ bg: vars.blueLightest, borderColor: vars.brandBlue })}
+    ${mixin.dropzoneStatus({ bg: "#f3f8fc", borderColor: "#0099cc" })}
   }
```

Comment:
yeah the hardcoded color values should be fine for now, we should think about how to manage again next time

Reply:
ya true